test/unit/css.js

```diff
@@ -1323,11 +1323,11 @@ QUnit.test( "Keep the last style if the new one isn't recognized by the browser
 	assert.equal( el.css( "position" ), "absolute", "The old style is kept when setting an unrecognized value" );
 	el = jQuery( "<div></div>" ).css( "position", "absolute" ).css( "position", " " );
 
-	// Support: Edge 14
+	// Support: Edge 14-15
```

Comment:
> That works for them, but is just not acceptable when it can introduce ambiguity. 1.0.0 - 1.1.0-3.1.4 is a valid version range, but take away the spaces and its meaning is no longer clear.

Looks ambiguous to me either way, I'm not sure what that means even.

> Those are fine opinions, and you can incorporate them in a proposed fix for jquery/contribute.jquery.org#95 if you like.

Yeah, I can do that too, in fact, I told you that many times, since we still following the rules which weren't documented in any way.

If we agree that this if fine opinions I can create a pull to make all the comments like correct and consistent